docs/getting_started/import_format.md

```diff
@@ -63,234 +83,173 @@ Using the project template generates already a `manifest.json` file in the proje
+![Plug-in management view](../../assets/images/getting_started/import_format/plugin_view_simpletxt.png "Plug-in management view"){: .bare }
 
-![Auto Importer import plan](../../assets/images/getting_started/import_format/import_plan_settings.png "Auto Importer import plan")
+When the plug-in is loaded and shows no errors, the new format is available and will automatically be used in all import plans. We can now try to import the example file by creating a new import plan (or reusing an existing one). Configure a source folder, a target *PiWeb* backend and hit the run button.
 
-Before you start the Auto Importer you can check whether the SimpleTxt format is listed as a new format in the import configuration. For this click on the `Configure` button in the `Settings` tab of your import plan. In the format list of the import configuration view should you find an entry for the SimpleTxt format.
+![Auto Importer import plan](../../assets/images/getting_started/import_format/import_plan_settings.png "Auto Importer import plan"){: .bare }
 
-![Auto Importer import configuration](../../assets/images/getting_started/import_format/import_configuration.png "Auto Importer import configuration")
+Now you can drop [SimpleTxt-Example.txt](https://raw.githubusercontent.com/ZEISS-PiWeb/PiWeb-Import-Sdk/refs/heads/develop/examples/FirstImportFormat/SampleData/SimpleTxt-Example.txt){:target="_blank"} in the configured import folder to import it. If everything worked correctly, the resulting import history will look similar to this:
 
-When the SimpleTxt format is listed in the import configuration dialog and the example file is placed in the import folder you can start the import plan by clicking the `Start` button in the Auto Importer. Then you can check whether the import of the SimpleTxt file was successful in the import history. Therefore click on the `Show history` link in the `Status` tab of the import plan. An new entry in the import history for the SimpleTxt format should be visible and should you inform whether the import was successful.
+![Auto Importer import history](../../assets/images/getting_started/import_format/import_history.png "Auto Importer import history"){: .bare }
 
-![Auto Importer import history](../../assets/images/getting_started/import_format/import_history.png "Auto Importer import history")
+You can also open *PiWeb Planner* and connect to the same *PiWeb backend*. A new measurement with measured values for the characteristics `CharA` and `CharB` should exist.
 
-You can also open a PiWeb Planner and connect to your PiWeb Cloud or PiWeb Server. A new measurement with measured values for the characteristics `CharA` and `CharB` should be exist.
+![Planner measurement view](../../assets/images/getting_started/import_format/planner_measurement.png "Planner measurement view"){: .bare }
```

Comment:
There is no {: .bare } css rule anymore. Its default now.

Reply:
Removed all {: .bare } declarations.